app/game/typingGame/styles.css

```diff
@@ -0,0 +1,52 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
```

Comment:
Good use of flexbox and min height! To integrate this with our app, we may need to add this to a div w/ absolute position and a z-index above the app to overlay this on top so min-height may not be necessary once we do that, but good use here